src/pages/IstioConfigDetails/IstioConfigDetailsPage.tsx

```diff
@@ -51,7 +51,11 @@ class IstioConfigDetailsPage extends React.Component<RouteComponentProps<IstioCo
 
   updateTypeFilter = () => this.updateFilters(true);
 
-  fetchIstioObjectDetails = (props: IstioConfigId) => {
+  fetchIstioObjectDetails = () => {
+    this.fetchIstioObjectDetailsFromProps(this.props.match.params);
```

Comment:
is match.params always non empty and won't "crash"?

Reply:
It shouldn't, since these props are not marked optional (thanks typescript for helping here!)